3_javascript/javascript1/week3/Homework/script.js

```diff
@@ -0,0 +1,219 @@
+// warm up work item array removal //
+
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+// Write some code here
+
+console.log(names.at(1));
+
+// trying other parameters //
+
+console.log(names.indexOf("Rasmus"));
+
+console.log(names[names.length - 1]);
+
+// Code done
+
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+
+//  When will we be there //
+
+const travelInformation = {
```

Comment:
you didn't use this const i your function to make a calculation

Reply:
I still don't understand this part here.